docs/kjar/README.md

```diff
@@ -0,0 +1,75 @@
+### Kjar
+
+#### Deploy kjar at startup from the fat jar
+
+This is the default configuration using the sample-hacep-project/sample-hacep-project-kjar maven module, 
+no env vars are required but it can't be updated at runtime.
+
+#### Deploy kjar at startup from Maven Repo
+
+To enable the update at startup and at runtime some ENV VArs are needed in the deployment.yaml
+and the dependency 
+```xml
+<dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>sample-hacep-project-kjar</artifactId>
+</dependency>
+```
+must be removed from you maven module,
+some other configuration are described in the following steps
+
+
+
+
+##### The prerequisites
+To have an updatable Kjar at startup and later, is mandatory to add two env vars 
+at the startup time and the presence of the specific jar in a Maven repo. 
+
+```yaml
```

Comment:
Please provide info about which yamls should be modified. For example links etc.

Reply:
Ok

Reply:
Seems that links to modules does not work. Probably missing slash at the beginning?